src/assets/styles/components/_chat.scss

```diff
@@ -0,0 +1,393 @@
+      border-color: rgb(224, 224, 224);
+      overflow-x: auto;
+      max-width: 100%;
+    }
+
+    code:before {
+      letter-spacing: inherit;
+    }
+    p {
+      margin-bottom: 0;
+    }
+    p:not(:last-child) {
+      margin-bottom: 8px;
+    }
+    a {
+      text-decoration: none;
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+    pre {
+      font-size: 12px;
+      padding: 8px;
+      margin-bottom: 8px;
+      tab-size: 4;
+      border-radius: 4px;
+      border-style: solid;
+      border-width: 1px;
+      overflow-x: auto;
+      max-width: 100%;
+      border-color: rgb(224, 224, 224);
+
+      code:before {
+        content: '';
+      }
+    }
+  }
+
+  /**
+   * 1. align text to top
+   */
+  &__sender {
+    font-size: map-get($chat-sender, 'font-size');
+    line-height: map-get($chat-sender, 'line-height');
+    font-weight: 700;
+    margin-right: 8px;
+  }
+  &__timestamp {
+    font-size: 10px;
+    line-height: 1;
+  }
+  &__amount {
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 1;
+    margin-top: 6px;
+    margin-bottom: 6px;
+    &--clickable {
+      cursor: pointer;
+    }
+  }
+  &__rates-column {
+    flex-direction: column;
+  }
+  &__rates {
+    color: hsla(0, 0%, 100%, 0.7);
+    font-style: italic;
+    font-size: 14px;
+    font-weight: 300;
+  }
+  &__status {
+    line-height: 1;
+    margin-left: 8px;
+
+    i {
+      line-height: 1.2;
+    }
+  }
+  &__blockchain-status {
+    font-size: 12px;
+    line-height: 1;
+    margin-right: 5px;
+  }
+  &__direction {
+    font-size: 14px;
+    font-weight: 300;
+    border-bottom: 1px solid map-get($adm-colors, 'secondary2');
+    padding-bottom: 5px;
+    margin-bottom: 15px;
+  }
+  &__form {
+    padding: 8px;
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+
+    /**
+     * Vuetify
+     * 1. fix Firefox `textarea` height.
+     * 2. reset textarea extra margins.
+     */
+    textarea {
+      height: 32px;
+    }
+    .v-text-field {
+      padding-top: 0;
+      margin-top: 0;
+    }
+    .v-text-field__slot {
+      font-weight: 400;
+    }
+  }
+  &__form-append {
+    margin-right: 8px;
+  }
+  &__fab {
+    position: absolute;
+    right: 16px;
+    bottom: 16px;
+    opacity: 0.7;
+    &:hover {
+      opacity: 1;
+    }
+  }
+}
+
+/** Themes **/
+.theme--light {
+  .a-chat {
+    &__content {
+      color: map-get($shades, 'black');
+    }
+    &__message {
+      background-color: #fff;
+      box-shadow: 0 1px 10px hsla(0, 0%, 39.2%, 0.06),
+        0 1px 1px hsla(0, 0%, 39.2%, 0.04),
+        0 2px 10px -1px hsla(0, 0%, 39.2%, 0.02);
+      &--red {
+        background-color: map-get($red, 'lighten-5');
+      }
+      &--green {
+        background-color: map-get($green, 'lighten-5');
+      }
+    }
+    &__sender {
+      color: map-get($grey, 'darken-4');
+    }
+    &__timestamp {
+      color: map-get($adm-colors, 'muted');
+    }
+    &__blockchain-status {
```

Comment:
use comma separated rule `&__child1, &__child2` because the properties are repeated

Reply:
no refactoring in this MR